README.md

```diff
@@ -108,7 +108,7 @@ It adds 6 shortcuts when editing a `Todo` file:
   "todo.timekeeping.finished.format": "YY-MM-DD HH:mm", // Format used for displaying time inside @done/cancelled
   "todo.timekeeping.elapsed.enabled": true, // Enable the @lasted/wasted tag
   "todo.timekeeping.elapsed.format": "short-compact", // Format used for displaying time diff inside @lasted/waster
-  "todo.timekeeping.estimate.format": "short-compact", // Format used for the `[est]` token
+  "todo.timekeeping.estimate.format": "short-compact", // Format used for the `[est]` and `[est-total]` token
```

Comment:
"token**s**" here too.

Reply:
Also you should update the list of settings in the readme too.

Reply:
> Also you should update the list of settings in the readme too.

Not sure what you mean 🤔